plugins/selectionhighlight.lua

```diff
@@ -56,3 +56,43 @@ function DocView:draw_line_body(line, x, y)
   return line_height
 end
 
+-- Special thanks to Adam(aethy)
+-- for writing code which served as a guide
+
+--- This added feature displays selection highlights in/on the scrollbar
+
+local old_draw_docview_scrollbar = DocView.draw_scrollbar
+
+function DocView:draw_scrollbar()
+  old_draw_docview_scrollbar(self)
+  local scrollbar = self.v_scrollbar
+
+  local mw = 5  --marker width
+  local mh = 8  -- marker height
+  local cmh = 4  --current selection marker height
+  local cm_color =  {common.color "#DA70D6"} --current selection marker color
```

Comment:
You're missing a `local common = require "core.common"`.

Reply:
oh sorry. will add "common".  it's added in mine locally.

yes marks don't expand with the scrollbar.